scrapper.py

```diff
@@ -17,47 +34,152 @@ class NumberOfArticlesOutOfRangeError(Exception):
 
 class IncorrectNumberOfArticlesError(Exception):
     """
-    Total number of articles to parse in not integer
+    Total number of articles to parse is not integer  # in|is !typo
```

Comment:
please, do not rewrite descriptions given

Reply:
Reverted. Will not happen again.